### PR TITLE
Ability to manipulate handler response from the parser

### PR DESCRIPTION
**Pull request by wilkerlucio**
This is an implementation idea, please let me know if there are preferred ways of doing it.

### CODE REVIEW

**Comment by wilkerlucio** [comment]
possible implementation for #17

File: src/untangled/server/impl/components/handler.clj [hunk 1]
```diff
@@ -82,6 +82,15 @@
...
     {} resp))
 
+(defn collect-response [data]
+  (let [response (atom {})]
+    (clojure.walk/prewalk
+      (fn [x]
+        (if-let [res (some-> x meta :untangled.server.core/with-response)]
+          (swap! response res))
+        x) data)
+    @response))
```
Comment (Pancia) [inline src/untangled/server/impl/components/handler.clj]: Does this need to be an atom? Couldn't it just as easily be a reduce?

Comment (wilkerlucio) [reply src/untangled/server/impl/components/handler.clj]: new version does that, thanks.

**Comment by awkay** [comment]
Looks great, thanks!

**Pull request merged**